sklearn/titanic/README.md

```diff
@@ -0,0 +1,138 @@
+## Introduction
+
+This sample code illustrates how to start a *GCE* instance and train a model using scikit-learn on the instance. We will be using the [Titanic dataset](https://www.kaggle.com/c/titanic) from [Kaggle](https://www.kaggle.com/). The goal is to train a classifier to predict whether a given passenger on Titanic survived or not.
+
+## Prerequisites
+
+Before we start, we need to:
+
+* login to [Kaggle](https://www.kaggle.com) and download the Titanic dataset, mainly [train.csv](https://www.kaggle.com/c/titanic/download/train.csv).
+
+* have a [*GCP*](https://cloud.google.com/) account, and [download](https://cloud.google.com/sdk/), install, and [configure](https://cloud.google.com/sdk/gcloud/reference/config/) the *Google Cloud SDK* on our local computer.
+
+* [create a project](https://cloud.google.com/sdk/gcloud/reference/projects/create) on *GCP* and set the project, region, and zone properties as instructed [here](https://cloud.google.com/sdk/gcloud/reference/config/set).
+
+* set these environment variables on the local computer:
+	* $MY_INSTANCE: the name of the new instance (e.g. titanic_trainer).
```

Comment:
@happyhuman : In the example, could you show them how to set the environment variables?

Instead of "(e.g. titanic_trainer)", could you do "(e.g. `MY_INSTANCE=titanic_trainer`)"?

Reply:
Good idea. I made the change.